src/Resources/skeleton/serializer/Normalizer.tpl.php

```diff
@@ -0,0 +1,30 @@
+<?= "<?php\n" ?>
+
+namespace <?= $namespace; ?>;
+
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
+
+class <?= $class_name ?> implements NormalizerInterface
+{
+    private $normalizer;
+
+    public function __construct(ObjectNormalizer $normalizer)
+    {
+        $this->normalizer = $normalizer;
+    }
+
+    public function normalize($object, $format = null, array $context = array()): array
+    {
+        $data = $this->normalizer->normalize($object, $format, $context);
+
+        // Here: add, edit, or delete some data
+
+        return $data;
+    }
+
+    public function supportsNormalization($data, $format = null): bool
+    {
+        return $data instanceof \App\Entity\BlogPost;
```

Comment:
I think you should update this line with $namespace/$class_name ?

Reply:
`$namespace\$class_name` === `App\Serializer\Normalizer\TopicNormalizer` (for exemple)

So no. Did I miss something?

Reply:
No. I read your PR too quickly ;) ok for me ;)